apps/E2E/src/Checkbox/pages/CheckboxPageObject.ts

```diff
@@ -21,45 +15,29 @@ class CheckboxPageObject extends BasePage {
     return await (await this._primaryComponent).isSelected();
   }
 
-  async waitForCheckboxChecked(timeout?: number): Promise<void> {
-    await browser.waitUntil(async () => await this.isCheckboxChecked(), {
-      timeout: timeout ?? this.waitForUiEvent,
-      timeoutMsg: 'The Checkbox was not toggled correctly',
-      interval: 1000,
-    });
```

Comment:
Let's add a comment here giving a quick explanation of the function (we should get in the habit of documenting each method even if it's seems redundant. It never hurts giving a brief explanation)

Reply:
done